packages/x-date-pickers/src/DesktopTimePicker/DesktopTimePicker.test.tsx

```diff
@@ -213,7 +213,12 @@ describe('<DesktopTimePicker />', () => {
       }),
     );
 
-    const shouldDisableTime: TimePickerProps<any>['shouldDisableTime'] = (value) => value === 10;
```

Comment:
This test is on the legacy picker.
We should probably add tests to the `describeValidation` instead.

Reply:
Added [tests to `testTextFieldValidation`](https://github.com/mui/mui-x/pull/7299/commits/028bec186dd88fbeb341a1e19176d60f9c7f893f)